torchvision/datasets/eurosat.py

```diff
@@ -0,0 +1,72 @@
+import os
+from typing import Any
+
+from .folder import ImageFolder
+from .utils import download_and_extract_archive
+
+
+class EuroSAT(ImageFolder):
+    """RGB version of the `EuroSAT <https://arxiv.org/pdf/1709.00029.pdf>`_ Dataset.
```

Comment:
I think it's best to link to the GitHub repo instead, which seems to be the "official" landing point for the dataset. The paper is linked from there.

```suggestion
    """RGB version of the `EuroSAT <https://github.com/phelber/eurosat>`_ Dataset.
```